.github/workflows/build.yml

```diff
@@ -48,3 +48,13 @@ jobs:
           coverage run --branch --source mmdeploy -m pytest -rsE tests/
           coverage xml
           coverage report -m
+      # Upload the coverage report for python3.7 && pytorch1.8.0
+      - name: Upload coverage to Codecov
+        if: ${{matrix.torch == '1.8.0' && matrix.python-version == '3.7'}}
+        uses: codecov/codecov-action@v1.0.10
```

Comment:
Saw different versions used in mmdet.
May need to check it to use `codecov/codecov-action@v2`.
https://github.com/open-mmlab/mmdetection/blob/fb5463e604d75d57d435128002a48b6a1759a6d9/.github/workflows/build.yml#L223

https://github.com/open-mmlab/mmdetection/blob/fb5463e604d75d57d435128002a48b6a1759a6d9/.github/workflows/build.yml#L150